adapter/protobuf/src/Mu/Adapter/ProtoBuf.hs

```diff
@@ -61,7 +61,8 @@ instance ProtoEnum Bool
 -- | Annotations for Protocol Buffers fields.
 data ProtoBufAnnotation
   = -- | Numeric field identifier for normal fields
-    ProtoBufId Nat
+    --   and whether it should be packed (only used for lists of number-like values)
+    ProtoBufId Nat Bool
     -- | List of identifiers for fields which contain a union
   | ProtoBufOneOfIds [Nat]
```

Comment:
Should this be changed to `ProtoBufOneOfIds [ProtobufId]`?

e.g. in

```proto
oneof string_or_int {
  string b = 2;
  int32 c   = 3;
}
```

one choice can be packed, the other cannot.

Reply:
Oh, I sort of assumed that if you have a list of `oneof`, then that cannot be packed. So we can have a list of `string_or_int` and have it packed? But how do you know then which element of the union you chose in each case?

Reply:
>A oneof cannot be repeated.

Huh! Didn't know that. So you can ignore my comment.

https://developers.google.com/protocol-buffers/docs/proto3#oneof-features

Reply:
One more argument for my "Protocol Buffers has the most subtleties ever" rant, to appear shortly xD